docs/install.md

```diff
@@ -0,0 +1,76 @@
+
+You can get the prebuilt binaries from our [release page](https://github.com/awslabs/soci-snapshotter/releases)
+or [build them from source](./build.md).
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Dependencies](#dependencies)
+- [Configure soci-snapshotter (optional)](#configure-soci-snapshotter-optional)
+- [Install soci-snpashotter for containerd with systemd](#install-soci-snpashotter-for-containerd-with-systemd)
+- [Config containerd](#config-containerd)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Dependencies
+
+soci-snapshotter has the following dependencies. Please follow the links or commands
+to install them on your machine:
+
+> We only mention the direct dependencies of the project. Some dependencies may
+> have their own dependencies (e.g., containerd depends on runc/cni). Please refer
+> to their doc for a complete installation guide (mainly containerd).
+
+- **[containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) >= 1.4** -
+required to run soci-snapshotter; to confirm please check with `sudo ctr version`.
+- **fuse** - used for mounting without root access (`sudo yum install fuse`).
+
+For fuse/zlib, they can be installed by your Linux package manager (e.g., `yum` or `apt-get`).
+
+## Configure soci-snapshotter (optional)
+
+Similar to containerd, soci-snapshotter has a toml config file which is located at
+`/etc/soci-snapshotter-grpc/config.toml` by default. If such a file doesn't exist,
+soci-snapshotter will use default values for all configurations.
+
+> Whenever you make changes to the config file, you need to stop the snapshotter
+> first before making changes, and restart the snpashotter after the changes.
+
+## Install soci-snpashotter for containerd with systemd
+
+If you plan to use systemd to manage your soci-snapshotter process, you can download
+the [`soci-snapshotter.service` unit file](../soci-snapshotter.service) in the
+repository root directory into `/usr/local/lib/systemd/system/soci-snapshotter.service`,
+and run the following commands:
+
+```shell
+sudo systemctl daemon-reload
+sudo systemctl enable --now soci-snapshotter
+```
+
+## Config containerd
+
+We need to configure and restart containerd to enable soci-snapshotter (this
+section assume your containerd is also managed by `systemd`):
+
+- Stop containerd: `sudo systemctl stop containerd`;
+- Update containerd config to include soci-snapshotter plugin. The config file
+is usually in `/etc/containerd/config.toml`, and you need to add the following:
+
+```toml
+[proxy_plugins]
+  [proxy_plugins.soci]
+    type = "snapshot"
+    address = "/run/soci-snapshotter-grpc/soci-snapshotter-grpc.sock"
+```
+
+- Restart containerd: `sudo systemctl restart containerd`;
+- (Optional) Check soci-snapshotter is recognized by containerd: `sudo ctr plugin ls id==soci`.
```

Comment:
nit: maybe we should note what the output should look like and mention that if the output is empty then something went wrong and you should consult the containerd logs to determine the cause.